utils/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# enable utils build a common library for all utils
+set(QUDA_UTILS_COMMON misc.cpp face_gauge.cpp host_utils.cpp host_blas.cpp command_line_params.cpp llfat_utils.cpp staggered_gauge_utils.cpp set_params.cpp)
+cuda_add_library(quda_utils STATIC ${QUDA_UTILS_COMMON})
```

Comment:
same comments for host_reference. No need for a variable here, use `add_library` directly.